### PR TITLE
Make model_id configurable for Llama2-c example 

### PR DESCRIPTION
**Pull request by EricLBuehler**
As stated above, this PR resolves a TODO comment by adding another command-line parameter to the eval command.

### CODE REVIEW

File: candle-examples/examples/llama2-c/main.rs [hunk 1]
```diff
@@ -178,10 +181,9 @@ fn run_eval(args: &EvaluationCmd, common_args: &Args) -> Result<()> {
     let tokens = match &args.pretokenized_dir {
         None => {
             let api = hf_hub::api::sync::Api::new()?;
-            let model_id = "roneneldan/TinyStories"; // TODO: Make this configurable.
```
Comment (EricLBuehler) [inline candle-examples/examples/llama2-c/main.rs:181]: Here was the TODO comment.

**Comment by EricLBuehler** [comment]
@LaurentMazare, what do you think of this PR? Do you think it is necessary?

**Comment by LaurentMazare** [comment]
> @LaurentMazare, what do you think of this PR? Do you think it is necessary?

I don't think this adds much and I would even think that it may break the existing code if I'm not mistaking. The dataset was coming from `roneneldan/TinyStories` and after that it seems to come from `karpathy/tinyllamas` which I wouldn't think has the data. Maybe I'm missing something here, not sure how much testing was done. Anyway i think it would be great if you want to transform this example into a full replication of Karpathy's awesome work on tinyllama but this likely belongs to an external repo. Happy to link to it if you put something into place that works well.

**Comment by EricLBuehler** [comment]
Ok, sounds good. I suppose the default value for `model_id` should have been `roneneldan/TinyStories`. Regardless, I will look into making an example of `tinyllama`. Thanks!

**Pull request closed without merging**